modules/setting/cors.go

```diff
@@ -19,10 +19,12 @@ var CORSConfig = struct {
 	Methods          []string
 	MaxAge           time.Duration
 	AllowCredentials bool
+	Headers          []string
 	XFrameOptions    string
 }{
 	Enabled:       false,
 	MaxAge:        10 * time.Minute,
+	Headers:       []string{"Authorization", "X-Gitea-OTP"},
```

Comment:
```suggestion
	Headers:       []string{"Authorization", "Content-Type", "User-Agent"},
```
This should be values you mention in the docs! And I think we should not add `X-Gitea-OTP` here because Gitea **needs** this header. It should not be configurable.